packages/components/src/icon/stories/index.story.tsx

```diff
@@ -47,26 +47,68 @@ FillColor.args = {
 	...Default.args,
```

Comment:
I don't see a control to add a classname anywhere in the stories. Should we have one?

Reply:
My thoughts on the `className` issue are [here](https://github.com/WordPress/gutenberg/pull/67282#discussion_r1858568220). I would prefer to align with the standard docgen behavior (i.e. allowed element props are not listed out unless explicitly highlighted).

Although if we get signals that the lack of `className` documentation is causing confusion, I'm also open to standardizing on "always document `className` explicitly".

Reply:
Too much documentation is rarely an issue, while too little docs has often been an issue. In favor of extenders, I'd personally suggest proactively documenting all props, even if they're general and available for every component. Not something to block this PR though.

Reply:
All props might be a bit too much, I agree. But useful props folks want to use most often make sense to be documented IMHO. 

> Can we find an alternative solution, maybe adding a paragraph where we explain that the component also accepts props X Y Z (or "every valid attribute and property for the X HTML element")?

That could work too 👍